packages/bpk-docs/src/pages/AlertPage/AlertPage.js

```diff
@@ -0,0 +1,139 @@
+import androidCancelable from 'react-native-bpk-component-alert/screenshots/android/cancelable.png';
+import androidThreeButton from 'react-native-bpk-component-alert/screenshots/android/three-button.png';
+
+import IntroBlurb from './../../components/neo/IntroBlurb';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import DocsPageWrapper from './../../components/neo/DocsPageWrapper';
+import Paragraph from './../../components/Paragraph';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotDefault}`,
+        altText: 'iOS Default Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotDefault}`,
+        altText: 'Android Default Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+  {
+    id: 'cancelable',
+    title: 'Cancelable',
+    blurb: [
+      <Paragraph>
+        Android alert can optionally be marked as cancelable. Clicking outside
+        the bounds of a cancelable alert will dismiss it without a callback.
+      </Paragraph>,
+    ],
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosCancelable}`,
+        altText: 'iOS Cancelable Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidCancelable}`,
+        altText: 'Android Cancelable Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+  {
+    id: 'three-button',
+    title: 'Three Buttons',
+    blurb: [
+      <Paragraph>
+        iOS alert can accomodate any number of buttons. Each button can
+        optionally specify a style, which is one of &apos;default&apos;,
+        &apos;cancel&apos; or &apos;destructive&apos;.
+      </Paragraph>,
+    ],
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosThreeButton}`,
+        altText: 'iOS Three button Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidThreeButton}`,
+        altText: 'Android Three button Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+];
+
+const AlertSubPage = () => (
+  <DocsPageBuilder
+    title="Alert"
+    components={components}
+    readme={readme}
+    wrapped
+  />
+);
+
+const AlertPage = () => (
+  <DocsPageWrapper
+    title="Alert"
+    blurb={[
+      <IntroBlurb>
+        Alerts are used to provide the user with a choice of actions. On android
```

Comment:
* `android` should be capitalised (`Android`).
* `colors` should use British English (`colours`).